assets/js/blocks/mini-cart/mini-cart-contents/inner-blocks/mini-cart-title-label-block/block.tsx

```diff
@@ -19,22 +15,12 @@ type Props = {
 };
 
 const Block = ( props: Props ): JSX.Element => {
-	const colorProps = useColorProps( props );
-	const typographyProps = useTypographyProps( props );
-	const spacingProps = useSpacingProps( props );
+	const styleProps = useStyleProps( props );
 
 	return (
 		<span
-			className={ classNames(
-				props.className,
-				colorProps.className,
-				typographyProps.className
-			) }
-			style={ {
-				...colorProps.style,
-				...typographyProps.style,
-				...spacingProps.style,
-			} }
+			className={ classnames( props.className, styleProps.className ) }
+			style={ { ...styleProps.style } }
```

Comment:
This could be simplified as well, if I'm not wrong:
```suggestion
			style={ styleProps.style }
```